src/value.rs

```diff
@@ -209,13 +209,13 @@ mod tests {
         "hello" => "yoyo"
       }
     });
-    assert_eq!(object.get(point![]), Some(object.clone()));
-    assert_eq!(object.get(point!["hello"]), Some(value!(true)));
-    assert_eq!(object.get(point!["yolo"]), Some(value!("swag")));
-    assert_eq!(object.get(point!["5"]), Some(value!()));
-    assert_eq!(object.get(point!["world", "hello"]), None);
-    assert_eq!(object.get(point!["moon", "hello"]), Some(value!("yoyo")));
-    assert_eq!(object.get(point!["moon", "nope"]), None);
+    assert_eq!(object.get(point![]).cloned(), Some(object.clone()));
```

Comment:
[`cloned`](http://doc.rust-lang.org/std/option/enum.Option.html#method.cloned) is the opposite of `as_ref`.